### ISSUE TITLE
Add landmarks to the Onboarding Wizard

### ISSUE THREAD

**Issue by afercia** [Yoast/yoast-components#258]
A simple and effective way to improve accessibility and allow assistive technologies users to easily jump through the Onboarding Wizard main sections would be adding landmarks (an dI don't know why I haven't realized this earlier 😬  ).

That is, _all_ content should be wrapped within elements that have either a native semantics of landmark regions or an equivalent aria role.

#256 is going to add a `<header>` element, and when a `<header>` element is a direct descendant of the body it does have a landmark semantics. I'd consider to add a nav, main, and footer (I guess this last one should be done on the PHP side):

![screen shot 2017-09-01 at 11 02 48](https://user-images.githubusercontent.com/1682452/29963798-60768afa-8f08-11e7-98f6-d9ff70132348.png)

**Comment by afercia** [comment]
Note: for the header and footer changes in HTML 5.1 I think we should add explicit aria roles:

https://www.w3.org/TR/html51/sections.html#the-header-element
> Assistive Technology may convey the semantics of the header to users when the nearest ancestor sectioning content or sectioning root element is the body element. This information can provide a hint to users as to the type of content. For example the role of the element, which in this case is "banner", can be announced by screen reader software when a user navigates to an header element that is scoped to the body element. User Agents may also provide methods to navigate to header elements scoped to the body element.

https://www.w3.org/TR/html51/sections.html#the-footer-element
Assistive Technology may convey the semantics of the footer to users when the nearest ancestor sectioning content or sectioning root element is the body element. This information can provide a hint to users as to the type of content. For example the role of the element, which in this case is "content information", can be announced by screen reader software when a user navigates to an footer element that is scoped to the body element. User Agents may also provide methods to navigate to footer elements scoped to the body element.

See also
https://www.marcozehe.de/2017/08/30/firefox-57-will-less-chatty-screen-readers-situations/

**Comment by afercia** [comment]
This would be the combined effect of the changes proposed here,, in #256, and https://github.com/Yoast/wordpress-seo/issues/7807:

![wizard landmarks ok](https://user-images.githubusercontent.com/1682452/29965595-7c7a413c-8f0e-11e7-86a0-acdfeb0d7ca0.png)
